simp_le.py

```diff
@@ -1672,15 +1673,15 @@ def test_it(self):
             # NB get_stats() would fail if file didn't exist
             self.assertEqual(initial_stats, self.get_stats(*files))
 
-            self.assertEqual(EXIT_REVOKE_OK, self._run(
-                '--server %s --revoke -f account_key.json -f full.pem' %
-                self.SERVER))
+            self.assertEqual(EXIT_REVOKE_OK, self._run([
+                "simp_le", "-v", "--server", (self.SERVER), "--revoke",
+                "-f", "account_key.json", "-f", "full.pem"]))
             # Revocation shouldn't touch any files
             self.assertEqual(initial_stats, self.get_stats(*files))
 
             # Changing SANs should trigger "renewal"
-            self.assertEqual(
-                EXIT_RENEWAL, self._run('%s -d le2.wtf:%s' % (args, webroot)))
+            args.extend(["-d", "le2.wtf:%s" % (webroot)])
```

Comment:
Rather than using extend here, I'd rather just concatinate the lists, i.e.

```python
self.assertEqual(EXIT_RENEWAL, self._run(args + ["-d", "le2.wtf%s" % webroot]))
```

Rationale: mutating the list in-place makes the code more brittle, as the order of thee tests affects what arguments get passed. The same goes for the other mutations.